payment-service/src/main/java/com/paymentservice/paymentservice/service/PaymentServiceImpl.java

```diff
@@ -0,0 +1,278 @@
+package com.paymentservice.paymentservice.service;
+
+import com.paymentservice.paymentservice.dto.request.CardRequest;
+import com.paymentservice.paymentservice.dto.request.ChargeRequest;
+import com.paymentservice.paymentservice.dto.request.CustomerChargeRequest;
+import com.paymentservice.paymentservice.dto.request.CustomerRequest;
+import com.paymentservice.paymentservice.dto.response.*;
+import com.paymentservice.paymentservice.exception.AlreadyExistsException;
+import com.paymentservice.paymentservice.exception.NotFoundException;
+import com.paymentservice.paymentservice.exception.PaymentException;
+import com.paymentservice.paymentservice.model.User;
+import com.paymentservice.paymentservice.repository.CustomerRepository;
+import com.paymentservice.paymentservice.service.interfaces.PaymentService;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.*;
+import com.stripe.model.Charge;
+import com.stripe.model.Customer;
+import com.stripe.model.Token;
+import com.stripe.param.CustomerCreateParams;
+import com.stripe.param.CustomerUpdateParams;
+import com.stripe.param.PaymentIntentConfirmParams;
+import com.stripe.param.PaymentIntentCreateParams;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.paymentservice.paymentservice.util.Messages.*;
+
+@Service
+@RequiredArgsConstructor
+public class PaymentServiceImpl implements PaymentService {
+    @Value("${stripe.key.secret}")
+    private String SECRET_KEY;
+    @Value("${stripe.key.public}")
+    private String PUBLIC_KEY;
+    private final CustomerRepository customerRepository;
+
+    public StringResponse charge(ChargeRequest request) {
+        Stripe.apiKey = SECRET_KEY;
```

Comment:
Ты в каждом методе в сервисе назначаешь переменной apiKey значение SECRET_KEY, может стоит ее один раз инициализировать например в инит методе
```suggestion
        @PostConstruct
    public void init() {
        Stripe.apiKey = SECRET_KEY;
    }
```
Или если тебе нужно в каждом методе, то наверное лучше делать так:
```suggestion
        RequestOptions requestOptions = RequestOptions.builder()
                    .setApiKey(SECRET_KEY)
                    .build();

            charge = Charge.create(params, requestOptions);
```